src/hotspot/cpu/x86/x86_64.ad

```diff
@@ -11306,6 +11306,31 @@ instruct convD2L_reg_reg(rRegL dst, regD src, rFlagsReg cr)
   ins_pipe(pipe_slow);
 %}
 
+instruct convF2HF_reg_reg(rRegI dst, regF src, regF tmp) %{
+  predicate(VM_Version::supports_evex());
+  effect(TEMP tmp);
+  match(Set dst (ConvF2HF src));
+  format %{ "evcvtps2ph $dst,$src" %}
+  ins_encode %{
+     __ evcvtps2ph($tmp$$XMMRegister, $src$$XMMRegister, 0x04, Assembler::AVX_128bit);
+     __ movdl($dst$$Register, $tmp$$XMMRegister);
+     __ movswl($dst$$Register, $dst$$Register);
+  %}
+  ins_pipe( pipe_slow );
+%}
+
+instruct convHF2F_reg_reg(regF dst, rRegI src) %{
+  predicate(VM_Version::supports_evex());
```

Comment:
The predicate could be removed as it is fully covered in the match_rule_supported.